adding ubuntu 20.04.1 rpi server deps (#14116)
trailofbits/algo: README.md

            ```bash
            sudo apt install -y --no-install-recommends python3-virtualenv
            ```

>             * RPi ( ubuntu server: install these packages in conjunction with above dependencies ):
>                 ```
>                 sudo apt install libffi-dev libssl-dev
>                 ```

Reviewer: I think this would be clearer and more consistent:
```suggestion
            On a Raspberry Pi running Ubuntu also install `libffi-dev` and `libssl-dev`.
```

Author: no problem, it is now added :grin:

thanks for the suggestion :pray:, and that does make it flow better :slightly_smiling_face:
